src/test/groovy/org/prebid/server/functional/testcontainers/PbsConfig.groovy

```diff
@@ -73,6 +74,7 @@ LIMIT 1
         ["adapters.generic.enabled"      : "true",
          "adapters.generic.endpoint"     : "$rootUri/auction".toString(),
          "adapters.generic.usersync.url" : "$rootUri/generic-usersync".toString(),
+         "adapters.generic.ortb-version" : "2.5",
```

Comment:
I'd say let's go with 2.6 by default.

Reply:
Also some tests work only on 2.5 and for them need a special spawned container with 2.6, so I think to leave it as 2.5